help/moderation/appeals.md

```diff
@@ -0,0 +1,52 @@
+# Appealing a moderation decision 
+
+It is possible to appeal the classification of an announced article or the decision to decline a submission. In some cases, we may require that a paper be accepted for publication in a conventional journal before an appeal is considered. Note that some works that have been published or accepted by a journal may still be declined at arXiv’s discretion.
+
+If you disagree with a [moderation](/help/moderation) decision, you may appeal for your desired outcome. The appeals process allows moderators time to reconsider a submission, but repeated appeals with no additional information cannot be considered.
```

Comment:
"but repeated appeals with no additional information cannot be considered." 
Would it help if we were more specific here? I.e., Do we accept repeat appeals? If we do, what type of information is required?

Reply:
we don't, but we do get them. People will try and re-appeal the same decisions a year later. The intent here was to try and have something to point to that says we don't reconsider this because there's no new info.